nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -1,42 +1,26 @@
 ---
 title: Azure Storage Persistence Configuration
 summary: Configuring Azure Storage as persistence
+reviewed: 2016-05-12
 tags:
 - Azure
 - Cloud
 - Persistence
 - Configuration
 ---
 
-The behavior of each persister can be controlled by specifying one of the respective configuration sections in the `app.config` and changing available properties, or using code API. The exact same settings are exposed via `app.config` file and code configuration API.
+In NServiceBus.Azure the behavior of the `AzureStoragePersister` can be controlled by working with the appropriate configuration section(s) in the `app.config` or by using the code API. Both approaches to configuring the persister can access the same configuration options.
```

Comment:
Is the doco page specifically for ASP configuration the best place to put the explanation?

I've added a bit more clarity to the start of the asp-6to1 upgrade document. Is that possibly a better place for it? 

If we were to do it in the configuration document then we should be doing it in every other document related to ASP. Doing that is an option. Probably best to use a callout (`INFO`?) at the top of every ASP page then.